File: app/src/main/java/com/imperium/academio/ui/adapters/helper/ItemViewHolder.java
package com.imperium.academio.ui.adapters.helper;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.constraintlayout.widget.ConstraintLayout;
import androidx.recyclerview.widget.RecyclerView;

import com.imperium.academio.R;

public class ItemViewHolder extends RecyclerView.ViewHolder {
    public final ImageView icon;
    public final TextView title;
    public final TextView text;
    public final ConstraintLayout constraintLayout;

    public ItemViewHolder(@NonNull View itemView) {
        super(itemView);
        icon = itemView.findViewById(R.id.material_item_icon);
        title = itemView.findViewById(R.id.material_item_title);
        text = itemView.findViewById(R.id.material_item_text);
        constraintLayout = itemView.findViewById(R.id.material_item_rvl);
    }
}

File: app/src/main/java/com/imperium/academio/ui/adapters/helper/ItemViewHolder.kt
package com.imperium.academio.ui.adapters.helper

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.constraintlayout.widget.ConstraintLayout
import androidx.recyclerview.widget.RecyclerView
import com.imperium.academio.R

class ItemViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @JvmField
    val icon: ImageView = itemView.findViewById(R.id.material_item_icon)

    @JvmField
    val title: TextView = itemView.findViewById(R.id.material_item_title)

    @JvmField
    val text: TextView = itemView.findViewById(R.id.material_item_text)

    @JvmField
    val constraintLayout: ConstraintLayout = itemView.findViewById(R.id.material_item_rvl)

}